main.go

```diff
@@ -213,8 +215,18 @@ func applyEnvironment(input string) (output string, err error) {
 	return buf.String(), nil
 }
 
-func setEnvironmentFile(config string) error {
-	fullPath := filepath.Join(config, "env.toml")
+func setEnvironmentFile(config string, targetEnv string) error {
+	var envFile = "env.toml"
```

Comment:
Doesn't look like this is ever modified, so probably should be `const envFile = "env.toml"` instead